aea/connections/scaffold/connection.py

```diff
@@ -78,3 +83,56 @@ async def receive(self, *args, **kwargs) -> Optional[Envelope]:
         :return: the envelope received, or None.
         """
         raise NotImplementedError  # pragma: no cover
+
+
+class MyScaffoldSyncConnection(BaseSyncConnection):
+    """Proxy to the functionality of the SDK or API."""
+
+    MAX_WORKER_THREADS = 5
+
+    connection_id = PublicId.from_str("fetchai/sync_scaffold:0.1.0")
+
+    def __init__(self, *args, **kwargs):
+        """
+        Initialize the connection.
+
+        The configuration must be specified if and only if the following
+        parameters are None: connection_id, excluded_protocols or restricted_to_protocols.
+
+        :param configuration: the connection configuration.
+        :param identity: the identity object held by the agent.
+        :param crypto_store: the crypto store for encrypted communication.
+        :param restricted_to_protocols: the set of protocols ids of the only supported protocols for this connection.
+        :param excluded_protocols: the set of protocols ids that we want to exclude for this connection.
+        """
+        super().__init__(*args, **kwargs)
+        raise NotImplementedError  # pragma: no cover
+
+    def main(self):
+        """Run main."""
```

Comment:
Its not clear from the docstring what should be implemented here.

Reply:
updated